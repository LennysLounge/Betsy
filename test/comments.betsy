# Hash marks the start of a comment
print 12 # this will print 12
print + 144 56 # this will print 200

# A hash mark has to be sperated by a space. Otherwise
# the '#' is part of the identifier
var i# int 23
print i# # only this is a comment