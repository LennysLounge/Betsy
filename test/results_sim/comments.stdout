12
200
23
